push_notification/apps/notification/api_services/notification.py

```diff
@@ -0,0 +1,49 @@
+import logging
+from ..services.notification_service import NotificationService
+from ..serializers.notification import NotificationSerializer
+from ..exceptions import NotificationNotFound
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationAPIService(object):
+
+    @classmethod
+    def create_notification(cls, data):
+        logger.info("NotificationAPIService.create_notification: payload={}".format(data))
+        serialized_data = NotificationSerializer(data=data)
+        serialized_data.is_valid(raise_exception=True)
+        notification = NotificationService().create(data)
```

Comment:
Serialiser validations can happen in the view layer. What do you think ?

Reply:
I think ideally, APIService instead of calling internal function directly, It should call restAPI of respective service in order to perform respective operation. Any thought?